src/napari_threedee/_backend/manipulator/napari_manipulator_backend.py

```diff
@@ -34,6 +35,7 @@ def __init__(
         self._viewer = viewer
         self._layer = layer
         self.is_dragging = False
+        self.clamp = False
```

Comment:
I think it would be nice to use a more clear name. Perhaps something like `clamp_to_layer_bbox`? or something similar?

Reply:
Done, I also added a comment (And note to docstring)